### ISSUE TITLE
Can not install wazuh on Centos 7

### ISSUE THREAD

**Issue by 12obin** [wazuh/wazuh#3336]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.9.0 | Wazuh component | local | Source | Centos 7 |

The error I got 
`    CC libwazuhext.so
cc: error: external/libffi/server/.libs/libffi.a: No such file or directory
make: *** [libwazuhext.so] Error 1`

What am I missing here?

TIA

**Comment by vikman90** [comment]
Hi @12obin,

There is an issue in the compilation procedure (Makefile). The installer is compiling the _libffi_ library, but the Wazuh's Makefile is defining a parameter that conflicts with the _libffi_'s Makefile.

I'm working on a fix right now, but you can patch the Makefile easily: https://github.com/wazuh/wazuh/blob/v3.9.0/src/Makefile#L629

Replace this line:
```
LIBFFI_LIB = $(EXTERNAL_LIBFFI)server/.libs/libffi.a
```
With:
```
LIBFFI_LIB = $(EXTERNAL_LIBFFI)$(TARGET)/.libs/libffi.a
```

Clean your project and retry to install.

This script should fix the issue and recompile:
```shell
cd src
sed -i 's,LIBFFI_LIB = $(EXTERNAL_LIBFFI)server/.libs/libffi.a,LIBFFI_LIB = $(EXTERNAL_LIBFFI)$(TARGET)/.libs/libffi.a,' Makefile
make clean
cd ..
sudo ./install.sh
```

Hope that helps you. Thank you very much for reporting this problem.

Best regards.

**Comment by 12obin** [comment]
It works, thanks.

**Comment by vikman90** [comment]
@12obin I've just merged this fix into branch 3.10 so it will appear in the next minor version.

Thank you again for your feedback.
Best regards.
